pkg/sm/sm.go

```diff
@@ -155,8 +153,9 @@ func New(ctx context.Context, cancel context.CancelFunc, e env.Environment, cfg
 		cfg:                 cfg,
 	}
 
-	smb.RegisterPlugins(plugins.NewCatalogFilterByVisibilityPlugin(interceptableRepository))
-	smb.RegisterPlugins(osb.NewStoreServiceInstancesPlugin(interceptableRepository))
+	smb.RegisterPlugins(osb.NewCatalogFilterByVisibilityPlugin(interceptableRepository))
+	smb.RegisterPluginsBefore(osb.CheckInstanceOwnerPluginName, osb.NewStoreServiceInstancesPlugin(interceptableRepository))
```

Comment:
we now make calling EnableMultitenancy mandatory - otherwise SM will panic on bootstrap ( not finding the checkInstanceOwner plugin). This defeats the purpose for having EnableMultitenancy as a separate method.